eslint.config.mjs

```diff
@@ -19,6 +19,7 @@ export default [
       "**/out",
       "**/poky",
       "**/.vscode-test",
+      "integration-tests/project-folder",
```

Comment:
I just realized `project-folder` only contains bitbake files, I think you wanted to add the `src` folder where all the ts files are located.

Reply:
No, after we run any build during integration tests, there are some .js files getting fetched in it.
We do want to lint the integration-tests/src (OK), and ignore the project-folder (OK).